.gitignore

```diff
@@ -1,26 +1,27 @@
 *.pyc
-afs/media/node_modules
```

Comment:
As part of this PR, it might be worth it to just ignore anything in the AFS directory.  That way other incoming PRs don't get things twisted and we end up with arches_for_science and afs

Reply:
I like it

Reply:
[658d826](https://github.com/archesproject/arches-for-science-prj/pull/1286/commits/658d826322cb882bff0a5ca2b30bcf582f22e8ec)